pinot-spi/src/main/java/org/apache/pinot/spi/config/table/UpsertConfig.java

```diff
@@ -75,6 +75,15 @@ public enum Strategy {
   @JsonPropertyDescription("Whether to preload segments for fast upsert metadata recovery")
   private boolean _enablePreload;
 
+  @JsonPropertyDescription("Whether to enable consistent view for upsert table")
+  private boolean _enableUpsertView;
+
+  @JsonPropertyDescription("Whether to enable batch refresh mode to keep consistent view for upsert table")
+  private boolean _enableUpsertViewBatchRefresh;
```

Comment:
imo this should be enabled by default. At present, this is more of a bugfix rather than a feature to be kept behind a config. It can be kept behind a config initially to be on the safer side but then we should enable it by default in the long run.